docs/walkthrough/walkthrough-istio.md

```diff
@@ -206,8 +206,7 @@ Tekton Pipelines should be up and running.
 Installing the latest Tekton Dashboard release is done by running the following command:
 
 ```bash
-curl -sL https://raw.githubusercontent.com/tektoncd/dashboard/master/scripts/release-installer | \
-   bash -s -- build latest --output /dev/stdout | istioctl kube-inject -f - | kubectl apply -f -
+curl -sL https://storage.googleapis.com/tekton-releases/dashboard/latest/tekton-dashboard-release.yaml | istioctl kube-inject -f - | kubectl apply -f -
```

Comment:
The installer works fine here, i would rather keep the installer version if possible. What issue did you encounter ?

Reply:
I got
```
Error: error unmarshaling JSON: json: cannot unmarshal string into Go value of type v1.TypeMeta
error: no objects passed to apply
```

It's because Ko needs to be installed for the script. Since we want to keep the installer, I could add Ko to the before you begin section

Reply:
ko is not needed, the installer will use curl (or wget) and sed. Are you running on linux, windows or mac ?

Reply:
I'm on a mac

Reply:
What is the output of just the installer please ?

curl -sL https://raw.githubusercontent.com/tektoncd/dashboard/master/scripts/release-installer | \
   bash -s -- build latest --output /dev/stdout

Reply:
```
curl -sL https://raw.githubusercontent.com/tektoncd/dashboard/master/scripts/release-installer | \
   bash -s b latest --output /dev/stdout
```
gives me
```
ko is required
```

Reply:
Ok it’s a bug, let’s use the release manifest until we fix it. 

Thanks !